code/scripts/espurna_utils/bsec.py

```diff
@@ -0,0 +1,58 @@
+import os
+
+
+def bsec_inject_patcher(env):
+    bsec_builder = next(
+        (
+            builder
+            for builder in env.GetLibBuilders()
+            if builder.name == "BSEC Software Library"
+        ),
+        None,
+    )
+
+    if bsec_builder is None:
+        return
+
+    # Allows `import espurna_utils` for external scripts, where we might not be within scons runtime
+    from SCons.Script import Move, Delete, Mkdir, Touch
```

Comment:
I had exactly this issue when I first attempted to use native primitives like `Touch` and `Delete´:-)